src/www/ui/template/upload_instructions.html.twig

```diff
@@ -79,5 +79,5 @@
   <em>
     {{ 'Configuring the Scheduler in the '| trans }}
   </em>
-  <a href="http://www.fossology.org/projects/fossology/wiki/Sched-20">{{ 'Scheduler documentation'| trans }}</a>.
+  <a href="https://github.com/fossology/fossology/wiki">{{ 'Scheduler documentation'| trans }}</a>.
```

Comment:
Does using this link makes more sense?
https://github.com/fossology/fossology/wiki/Job-Scheduler

Reply:
okay @GMishx i'll update it with this